core/src/main/resources/application.conf

```diff
@@ -115,13 +115,13 @@ google {
   ]
 }
 
-engine {
-  filesystems {
-    gcs {
-      auth = "application-default"
-    }
-  }
-}
+//engine {
```

Comment:
Maybe leave the `engine` block in, and just have the `filesystems` commented out?